assets/cases/ui/12.editbox/editbox-ctrl.ts

```diff
@@ -1,32 +1,39 @@
-import { _decorator, Component, Node, EditBox } from "cc";
-const { ccclass, property, menu } = _decorator;
-
-@ccclass("EditboxCtrl")
-@menu('UI/EditboxCtrl')
-export class EditboxCtrl extends Component {
-
-    @property(EditBox)
-    public editBox1: EditBox = null!;
-    @property(EditBox)
-    public editBox2: EditBox = null!;
-    @property(EditBox)
-    public editBox3: EditBox = null!;
-
-    start () {
-        // Your initialization goes here.
-    }
-
-    setFocus (event: EditBox, custom: string) {
-        if(custom === '1'){
-            this.editBox1.setFocus();
-        } else if (custom === '2') {
-            this.editBox2.setFocus();
-        } else if (custom === '3') {
-            this.editBox3.setFocus();
-        }
-    }
-
-    // update (deltaTime: number) {
-    //     // Your update function goes here.
-    // }
-}
+import { _decorator, Component, Node, EditBox, screen, macro } from "cc";
+const { ccclass, property, menu } = _decorator;
+
+@ccclass("EditboxCtrl")
+@menu('UI/EditboxCtrl')
+export class EditboxCtrl extends Component {
+
+    @property(EditBox)
+    public editBox1: EditBox = null!;
+    @property(EditBox)
+    public editBox2: EditBox = null!;
+    @property(EditBox)
+    public editBox3: EditBox = null!;
+
+    start () {
+        // Your initialization goes here.
+        screen.on("orientation-change", (orientation) => {

```

Comment:
Should turn off it when switch to other test cases?

Reply:
Yes, I'll move it to a new test-case, and display the changes by ui

Reply:
updated